### PR TITLE
Fix for Dark theme not readable

### LINKED ISSUE #7927: Dark Theme : Search -> Import entries dialog text not readable
Latest JabRef version from main.

1. Have Dark Theme enabled 
1. Search for an entry in the web search
2. Import entries window opens
3.  See the color of the abstract text is black 

Bildschirmfoto 2021-07-19 um 21 27 49

### PR DESCRIPTION
**Pull request by victordmp**
### Initial Proposed

* Change the text color of the summary that was previously black. In order to improve the contrast in dark mode theme.

### My Proposed

* Change the color of the abstract text from black to a lighter color. In order to improve the contrast in the dark mode theme.
* Change the background color of `.list-cell:entry-selected` to a darker color. In order to improve the contrast with the color of the title and abstract.

#### In the dialog for importing entries, both the title and the summary value were difficult to read in Dark Theme mode. To solve this problem I did the following

* Fixed the background of the text selected in the `ImportEntriesDialog`.

   * **Before:**
        Screen Shot 2022-06-26 at 13 40 29
   * **After:**
        Screen Shot 2022-06-26 at 13 33 57

* Corrected the unselected text color of the summary in the `Import Entries Dialog`.

   * **Before:**
        Screen Shot 2022-06-26 at 13 33 57
   * **After:**
        Screen Shot 2022-06-26 at 13 37 53

### Checking the changes

1. Have Dark Theme enabled
2. Search for an entry in the web search
3. Import entries window opens

### PR requirements

- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/gui/Base.css [hunk 1]
```diff
@@ -98,6 +98,10 @@
     -jr-drag-target: -jr-purple;
     -jr-drag-target-hover: derive(-jr-purple, 80%);
 
+    /* For summary text color */
```
Comment (koppor) [inline src/main/java/org/jabref/gui/Base.css:101]: This comment can be removed (as it is nearly verbatim the name of the CSS class)

File: src/main/java/org/jabref/gui/importer/ImportEntriesDialog.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .addEntryButton {
-    -fx-font-size: 2em;
+    -fx-font-size: 35;
```
Comment (koppor) [inline src/main/java/org/jabref/gui/importer/ImportEntriesDialog.css:2]: Is there a reason why this was changed?

File: src/main/java/org/jabref/gui/importer/ImportEntriesDialog.css [hunk 3]
```diff
@@ -11,10 +11,15 @@
...
 
-.list-cell {
+.list-cell .summary > Text {
     -fx-padding: 0.5em 0 1em 0.5em;
+    -fx-fill: -js-summary-text-color;
 }
 
-.list-cell:entry-selected {
-    -fx-background-color: derive(-jr-selected, 60%);
+.list-cell:entry-selected{
```
Comment (koppor) [inline src/main/java/org/jabref/gui/importer/ImportEntriesDialog.css:19]: Please keep the space between the class name and `{`.

File: src/main/java/org/jabref/gui/importer/ImportEntriesDialog.css [hunk 4]
```diff
@@ -11,10 +11,15 @@
...
+    -fx-background-color: derive(-jr-selected, 35%);
+}
+
+.list-cell:entry-selected .summary > Text{
```
Comment (koppor) [inline src/main/java/org/jabref/gui/importer/ImportEntriesDialog.css:23]: Please keep the space between the class name and `{`.

**Review by koppor — CHANGES_REQUESTED** [review]
Thank you for the PR! I am always using the light theme. I switched to the dark theme - and it looks OK enough.

For future work, we should try to look at the color theme of Visual Studio Code. It has a very decent dark mode. (This also refs the discussion at https://github.com/JabRef/jabref/pull/7825#issuecomment-862355084)

File: src/main/java/org/jabref/gui/importer/ImportEntriesDialog.css (hunk 2 above)
Comment (victordmp) [reply src/main/java/org/jabref/gui/importer/ImportEntriesDialog.css]: Not really, it was something left over from some tests I was doing earlier. I forgot to change it again. Thanks for remembering.

**Comment by victordmp** [comment]
> Thank you for the PR! I am always using the light theme. I switched to the dark theme - and it looks OK enough.
> 
> For future work, we should try to look at the color theme of Visual Studio Code. It has a very decent dark mode. (This also refs the discussion at [#7825 (comment)](https://github.com/JabRef/jabref/pull/7825#issuecomment-862355084))

Thanks for the feedback. Threads solved.

**Comment by Siedlerchr** [comment]
Thanks, lgtm! Can you please add changelog entry as well?

**Comment by victordmp** [comment]
> Thanks, lgtm! Can you please add changelog entry as well?

Changelog updated.

**Comment by victordmp** [comment]
> Thanks again for the quick follow up!

You are welcome :)

**Pull request merged**